src/npctalk.cpp

```diff
@@ -672,7 +670,7 @@ std::string dialogue::dynamic_line( const std::string &topic ) const
                 beta->name.c_str());
     }
 
-    const auto p = beta; // for compatibility, later replace it in the code below
+    const auto &p = beta; // for compatibility, later replace it in the code below
```

Comment:
Nitpicking: It (`beta`) is a pointer anyway. Now you have `p` as a reference to a pointer, not really much more useful than a copy of the pointer. It's not wrong, it's probably not even bad, but that change will probably not have any effect at all. (One exception: if somehow the `beta` is changed to point to something else, `p` will no follow that new pointer, but that's an unlikely scenario to begin with.)